include/site-health.php

```diff
@@ -29,7 +29,7 @@ function notice_site($url, $check_health=false)
     $entry = $result[0];
     
     //If we are allowed to do health checks...
-    if(!!$check_health){
+    if($check_health){
```

Comment:
There are some standards "violations" in the text. Do we want to ignore them just because of the huge amount of changes?